demo/common/assets.js

```diff
@@ -1428,16 +1428,19 @@ shakaAssets.testAssets = [
       /* source= */ shakaAssets.Source.MICROSOFT)
       .addFeature(shakaAssets.Feature.MSS)
       .addFeature(shakaAssets.Feature.HIGH_DEFINITION)
-      .addFeature(shakaAssets.Feature.MP4),
+      .addFeature(shakaAssets.Feature.MP4)
+      .setMimeType('application/vnd.ms-sstr+xml'),
```

Comment:
This asset seems to have the correct headers, and it plays without problem, so I don't think it needs to be manually tagged as being MSS.
Unless you're worried about it changing.

Reply:
Done!